src/DurableTask.Netherite/Tracing/EtwSource.cs

```diff
@@ -378,11 +378,11 @@ public void FasterStorageProgress(string Account, string TaskHub, int PartitionI
             this.WriteEvent(265, Account, TaskHub, PartitionId, Details, AppName, ExtensionVersion);
         }
 
-        [Event(266, Level = EventLevel.Verbose, Version = 1)]
-        public void FasterAzureStorageAccessCompleted(string Account, string TaskHub, int PartitionId, string Intent, long Size, string Operation, string Target, double Latency, int Attempt, string AppName, string ExtensionVersion)
+        [Event(266, Level = EventLevel.Verbose, Version = 3)]
+        public void FasterAzureStorageAccessCompleted(string Account, string TaskHub, int PartitionId, string Intent, long Position, long Size, string Operation, string Target, double Latency, int Attempt, string AppName, string ExtensionVersion)
```

Comment:
```suggestion
        [Event(266, Level = EventLevel.Verbose, Version = 2)]
        public void FasterAzureStorageAccessCompleted(string Account, string TaskHub, int PartitionId, string Intent, long Position, long Size, string Operation, string Target, double Latency, int Attempt, string AppName, string ExtensionVersion)
```

Should this have been Version=2?

Reply:
just in case you already ran this in the cloud I decided to bump the version again

Reply:
Oh I didn't run it on Azure yet